### PR TITLE
Integrated toolset description now matches bionic abilities

### PR DESCRIPTION
**Pull request by ghost**
#### Summary
```SUMMARY: None```

#### Purpose of change
Integrated toolset doesn't have the wood saw or bolt cutter functionality, despite its description.

#### Describe the solution
Removed mention of wood saw and bolt cutters from bionic description.
#### Describe alternatives you've considered
Buffing the toolset to have those functions.

### CODE REVIEW

File: data/mods/Aftershock/recipes/afs_armor_recipes.json [hunk 1]
```diff
@@ -112,7 +112,7 @@
     "difficulty": 4,
     "time": 60000,
     "book_learn": [ [ "textbook_armwest", 3 ] ],
-    "tools": [ [ [ "cordless_drill", 20 ] ] ],
+    "tools": [ [ [ "cordless_drill", 20 ], [ "toolset", 30] ] ],
```
Comment (LaVeyanFiend) [inline data/mods/Aftershock/recipes/afs_armor_recipes.json]:
```suggestion
    "tools": [ [ [ "cordless_drill", 20 ], [ "toolset", 30 ] ] ],
```

**Comment by kevingranade** [comment]
Fix the description instead, simply having things in item descriptions never justify game changes.

**Comment by Rail-Runner** [comment]
Chopping down a tree with a wood saw which fits inside a hand is indeed a bit too much, in my opinion. 3 and 4 make sense to me, though, as those tools seem like they could be portable enough.

**Comment by ZhilkinSerg** [comment]
Integrated toolset can not be a full replacement for a toolbox.

**Comment by LaVeyanFiend** [comment]
Description changed to current abilities, only possible thing that's wierd now is the drill quality only being level 1(hand drill), rather than 3(cordless) when the bionic can certainly use power for functions already.

**Comment by ifreund** [comment]
Honestly I think it should have `DRILL 3`.  Having a super sophisticated bionic that has a mechanical hand drill makes little sense

**Comment by Amoebka** [comment]
I think we should consider what was the in-universe use for the integrated toolset when adjusting the tool qualities.

To me it would seem like such a CBM would be designed for handymen and people doing various repair jobs. If this is indeed the case, then high drilling ability should be included, while tree sawing definitely shouldn't.

**Comment by BenlyAZ** [comment]
If it's meant for handymen, either it should have level 2 wood sawing or the level 2 wood sawing requirement on some basic carpentry tasks (build a crate, put up a door, craft a table, some others) should be reconsidered.

**Pull request merged**